scipy/spatial/tests/test_hausdorff.py

```diff
@@ -123,3 +123,29 @@ def test_invalid_dimensions(self):
         B = np.random.rand(4, 5)
         with pytest.raises(ValueError):
             directed_hausdorff(A, B)
+
+    @pytest.mark.parametrize("A, B, seed, expected", [
+        # the two cases from gh-11332
+        ([(0,0)],
+         [(0,1), (0,0)],
+         0,
+         (0.0, 0, 1)),
+        ([(0,0)],
+         [(0,1), (0,0)],
+         1,
+         (0.0, 0, 1)),
+        # slightly more complex case
+        ([(-5, 3), (0,0)],
+         [(0,1), (0,0), (-5, 3)],
+         77098,
+         # the maximum minimum distance will
+         # be the last one found
```

Comment:
I don't think the CI failures are related, but this statement had me thinking after I wrote it last night, given the shuffling of the data sets. That said, it is slightly different than the originally-reported issue, for which there is a single unique solution.

Reply:
Yes, indeed! The uniqueness of the solution is not guaranteed in general either, even when `u` is not a subset of `v`.  Because of possible symmetries in both `u` and `v`, multiple pairs of points could give the same directed Hausdorff distance.

Reply:
Ok, that makes sense I think. It is perhaps "ok" to test that for a given random seed the answer is fixed.

Reply:
I've extended the comment slightly to clarify this.